.gitignore

```diff
@@ -25,8 +25,11 @@ wheelhouse
 *.whl
 *.venv
 
-#Model artifacts
+# Model artifacts
 *.pt
 *.safetensors
 *.gguf
 *.vmfb
+
+# Tuner artifacts
+tuning_*/
```

Comment:
I'd prefer to put this in `tuner/.gitignore`, with the artifacts similarly scoped, so each subproject is more self-contained. As we're setting up packaging for each subproject I don't want the line between packages to blur like this.

Reply:
The thing with the current module setup is that it needs to start at the root of the project: `python -m tuner.examples.dispatch tuner/examples/dispatch/benchmark.mlir --num-candidates=20`, so this guards against any dirs created by running examples.

Maybe there's a better way to set up the module path, but I don't know how to do it.

Reply:
One option is to put a nested `SHARK-Platform/tuner/tuner/` directory, as with https://github.com/nod-ai/SHARK-Platform/tree/main/sharktank/sharktank and https://github.com/nod-ai/SHARK-Platform/tree/main/shortfin/python/shortfin

Reply:
OK, let me try that instead...

Reply:
That's top of my mind since https://github.com/nod-ai/SHARK-Platform/pull/244#pullrequestreview-2344357447

> Python projects have an annoying way of repeating primary directory names...